test/migration/subTests.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { BigNumber } from "@ethersproject/bignumber";
```

Comment:
Is this change in the scope of this PR?

Reply:
yes ... somehow ethers was complaining because it pulled different lib version .. not sure why ... #jsthings

Reply:
Weird, the lock file only has a single `@ethersproject/BigNumber` version:

https://github.com/gnosis/safe-contracts/blob/9b305a0f80da7f1107d1181f52c844f089557d05/yarn.lock#L197-L204